app/common/auth/login.controller.js

```diff
@@ -1,7 +1,8 @@
 module.exports = LoginController;
 
-LoginController.$inject = ['Authentication','$location'];
-function LoginController(Authentication, $location) {
+LoginController.$inject = ['Authentication','$location', 'Features'];
+function LoginController(Authentication, $location, Features) {
     Authentication.openLogin();
-    $location.url('/');
+    var path = Features.isFeatureEnabled('private') ? '/private' : '/';
```

Comment:
This is checking if the deploy is allowed to make the deployment private, not if the deploy IS private. Very different things.